ros_gz_bridge/launch/ros_gz_bridge.launch.py

```diff
@@ -46,12 +47,18 @@ def generate_launch_description():
         description='Name of container that nodes will load in if use composition',
     )
 
+    declare_create_own_container_cmd = DeclareLaunchArgument(
+        'create_own_container',
+        default_value='False',
+        description='Whether to start a ROS container when using composition',
```

Comment:
I haven't checked if the formatting is okay, but I suggest adding more documentation here and saying this is not recommended.

```suggestion
        description='Whether the bridge should starts its own ROS container when using composition (Not Recommended). \
         This option should only be set to true if the you plan to put your ROS node in the container created by the bridge. \
          This is not needed if you want Gazebo and the bridge to be in the same ROS container. ',
```